packages/cubejs-schema-compiler/adapter/PostgresQuery.js

```diff
@@ -29,6 +29,19 @@ class PostgresQuery extends BaseQuery {
   timeGroupedColumn(granularity, dimension) {
     return `date_trunc('${GRANULARITY_TO_INTERVAL[granularity]}', ${dimension})`;
   }
+
+  hllInit(sql) {
+    return `hll_add_agg(hll_hash_any(${sql}))`;
+  }
+
+  hllMerge(sql) {
+    return `hll_cardinality(hll_union_agg(${sql}))`;
+    return sql;
+  }
+
+  countDistinctApprox(sql) {
+    return  `hll_cardinality(${sql})` 
```

Comment:
This seems incorrect. Could you please test it?

Reply:
This worked for our test schema.  I will change it for `hll_cardinality(hll_add_agg(hll_hash_any(${sql})))` .